File: packages/aurora/src/main/java/com/sofar/aurora/model/Banner.java
package com.sofar.aurora.model;

import com.google.gson.annotations.SerializedName;

public class Banner {

  @SerializedName("title")
  public String title;

  @SerializedName("pic")
  public String imageUrl;

  @SerializedName("jumpLinkOutput")
  public String jumpLinkOutput;

  @JumpType
  @SerializedName("jumpType")
  public String jumpType;

  public @interface JumpType {
    String TYPE_ALBUM = "2";
    String TYPE_TRACK = "3";
  }

}

File: packages/wan-android/src/main/java/com/sofar/wan/android/model/Banner.kt
package com.sofar.wan.android.model

import com.google.gson.annotations.SerializedName

class Banner {

  @SerializedName("id")
  var id: Long = 0

  @SerializedName("title")
  var title: String = ""

  @SerializedName("desc")
  var desc: String = ""

  @SerializedName("imagePath")
  var imageUrl: String = ""

  @SerializedName("url")
  var url: String = ""
}

class Banners(
  val banners: List<Banner>
)